packages/cli/src/commands/proxy.ts

```diff
@@ -1,8 +1,9 @@
 import { pick } from 'lodash';
-import { CommandModule } from 'yargs';
+import { CommandModule, parsed } from 'yargs';
 import { createMultiProcessPrism, CreateProxyServerOptions, createSingleProcessPrism } from '../util/createServer';
 import sharedOptions from './sharedOptions';
 import { runPrismAndSetupWatcher } from '../util/runner';
+import { boolean } from 'fp-ts';
```

Comment:
is this import actually used?

Reply:
nope - I am not sure how that slipped in. Removed now.